randovania/games/samus_returns/logic_database/Surface - East.txt

```diff
@@ -368,8 +368,9 @@ Extra - asset_id: collision_camera_008
       Trivial
   > Door to Charge Beam Shaft (Upper)
       Any of the following:
-          Super Jump (Beginner) or Climb Rooms Vertically (High Jump)
+          Super Jump (Beginner) or Climb Rooms Vertically (High Jump) or Use Spider Ball
           Morph Ball and Unmorph Extend (Intermediate)
+          Infinite Bomb Jump (Beginner) and Simple IBJ
```

Comment:
The template already accounts for those options
![image](https://github.com/randovania/randovania/assets/38679103/e907d701-dc1b-4caa-8956-05737e11b84f)

Reply:
ah okay, ill close this PR then, i probably should have looked at the templates first